docs/getting-started-kubernetes.rst

```diff
@@ -16,7 +16,1259 @@
 
 .. headings # #, * *, =, -, ^, "
 
+.. contents:: On this page
```

Comment:
This will render TOC better:

```
.. headings # #, * *, =, -, ^, "


*******************************
Getting Started with Kubernetes
*******************************

.. contents:: On this page
   :depth: 2
   :local:
   :backlinks: none
```